src/core/reactor.cc

```diff
@@ -2218,6 +2219,36 @@ void reactor::kill(pid_t pid, int sig) {
     ret.throw_if_error();
 }
 
+future<std::optional<struct group>> reactor::getgrnam(std::string_view name, char *buf, size_t buflen) {
+    return futurize_invoke([name, buf, buflen, this] {
```

Comment:
https://github.com/scylladb/seastar/pull/2314#discussion_r1665946683 is not addressed. could you reconsider it?